app/assets/javascripts/discourse/app/components/sidebar/section-link.js

```diff
@@ -1,3 +1,13 @@
 import Component from "@ember/component";
+import discourseComputed from "discourse-common/utils/decorators";
+import { htmlSafe } from "@ember/template";
 
-export default Component.extend({});
+export default Component.extend({
```

Comment:
We're using Glimmer components going forward so let us change this to a Glimmer component.